<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NGBLR-03222" failures="2" tests="2" name="Default test(failed)" time="23.661" errors="0" timestamp="2023-10-16T20:48:14 IST">
  <testcase classname="contactsTests.CreateContactTest" name="createContactTest" time="5.364">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contactsTests.CreateContactTest.createContactTest(CreateContactTest.java:86)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createContactTest -->
  <testcase classname="contactsTests.CreateContactTest" name="demo" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contactsTests.CreateContactTest.demo(CreateContactTest.java:120)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
</testsuite> <!-- Default test(failed) -->
